frontend/src/redux/Account.ts

```diff
@@ -0,0 +1,31 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+/**
+ * Due to serialization issues, we can't directly store the
+ * firebase.User object in Redux (instead we store a toJson
+ * object version). This means we'll have to manually specify
+ * the type and copy any relevant fields from the docs below:
+ * https://firebase.google.com/docs/reference/js/firebase.User
+ */
+export type AccountType = {
```

Comment:
Would additional account attributes like access level go here as well, and be linked to the SQL database?

Reply:
I think this is only for the attributes provided in Firebase's user object - in the next PR though I expanded/refactored this redux state to include both the firebase user and also a custom account type, which would be linked to our backend database.